einstein/forward.py

```diff
@@ -87,58 +98,70 @@ async def run_step(
 
     log_event(self, event)
 
-    return event
+    return event, top_response
 
 
 async def forward(self):
     bt.logging.info("🚀 Starting forward loop...")
 
     while True:
-        bt.logging.info(
```

Comment:
Cannot remove this entirely, these must be running in parallel, 1 to mimic human tasks for the miner to develop the model, and 1 is for end to end user to use